modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutField.tsx

```diff
@@ -174,9 +173,8 @@ export default function ModalAddObjectLayoutField({
 								</ClayLabel>
 
 								{Liferay.FeatureFlags['LPS-170122'] &&
-									(readOnlyField.value === 'true' ||
-										readOnlyField.value ===
-											'conditional') && (
+									(readOnlyField === 'true' ||
```

Comment:
Hi @danvb112 everything works fine in my tests. Could you just order this verification by alphabetical order? Like 
`readOnlyField === 'conditional' || readOnlyField === 'true'`. Thanks :) 

Please apply that change in the other places that occurs as well.